packages/react-component-library/src/components/DatePicker/Month.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { useMonth, FirstDayOfWeek } from '@datepicker-react/hooks'
+import { IconChevronLeft, IconChevronRight } from '@royalnavy/icon-library'
+import { Day } from './Day'
+import { NavButton } from './NavButton'
+
+export interface MonthProps extends ComponentWithClass {
+  year: number
+  month: number
+  firstDayOfWeek: FirstDayOfWeek
+  goToPreviousMonths: () => void
+  goToNextMonths: () => void
+}
+
+const formatDayLabel = (dayLabel: string): string => {
+  switch (dayLabel) {
```

Comment:
Switch statements go in the same category as `else if else if else if` 😄 

Another option is:

```
const dayFormat = {
  'Mo': 'Mon',
  'Tu': 'Tue',
  'We': 'Wed',
  'Th': 'Thu',
  'Fr': 'Fri',
  'Sa': 'Sat',
  'Su': 'Sun',
}

const formatDayLabel = (dayLabel: string): string => {
  return dayFormat[dayLabel] || dayLabel
}
```

Reply:
😂